docs/tutorials/testing_with_control_center.rst

```diff
@@ -3,33 +3,27 @@
 
 .. _control_center_tutorial:
 
-*********************************************************
-Getting Started with Speed Motors Using IQ Control Center
-*********************************************************
+************************************************************
+Getting Started with Speed Modules Using IQ Control Center
+************************************************************
 
-The easiest way to get started with any Vertiq Motor is with the IQ Control Center. 
+The easiest way to get started with any Vertiq module is with the IQ Control Center. 
 The IQ Control Center is a simple application that allows you to configure and test
-a Vertiq Motor from your PC, without the need for a flight controller or specialized hardware.
+a Vertiq module from your PC, without the need for a flight controller or specialized hardware.
 
-This tutorial will take you through the process of getting started with a motor using
+This tutorial will take you through the process of getting started with a module using
 the IQ Control Center. It will outline how to setup the necessary hardware and software,
 provide a brief introduction to the Control Center, and teach you how to peform some basic
-tests on the motor with the Control Center.
+tests on the module with the Control Center.
 
-**This tutorial is applicable for both the Vertiq 2306 and the Vertiq 8108, but only when 
-using the Speed firmware. If you are using the position firmware, some of the parameters discussed here
-will not be available.**
+**This tutorial was created using a Vertiq 8108 module, on firmware version 0.0.7 and with Control Center version 1.3.4. However, this tutorial is applicable for any Vertiq module when using
```

Comment:
... created using a Vertiq 8108 module on firmware version 0.0.7, and with Control Center ...